src/components/Person/Person.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export function Person({ person }) {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerType = sex === `m` ? `wife's` : `husband's`;
+  const partner = isMarried
+    ? `My ${partnerType} name is ${partnerName}`
+    : `I am not married`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
```

Comment:
you should use conditional rendering for the whole p element, it's no need to have an empty p